GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/ScourgeHelper.cs

```diff
@@ -21,6 +24,17 @@ internal static class ScourgeHelper
             new BuffGainCastFinder(SandstormShroudSkill, DesertShroudBuff).UsingDurationChecker(3500),
             // new EXTBarrierCastFinder(DesertShroud, DesertShroud),
             new EXTBarrierCastFinder(SandCascadeSkill, SandCascadeBarrier),
+            new BuffGainCastFinder(SadisticSearing, SadisticSearing),
+            new BuffLossCastFinder(SadisticSearingActivation, SadisticSearing).UsingChecker((blcf, combatData, agentData, skillData) => 
+            {
+                AbstractBuffEvent abe = combatData.GetBuffData(SadisticSearing).Where(buffEvent => buffEvent is BuffApplyEvent).LastOrDefault(x => x.Time < blcf.Time);
+                AbstractHealthDamageEvent ahde = combatData.GetDamageData(ManifestSandShadeShadeHit).LastOrDefault(x => x.Time < blcf.Time);
```

Comment:
Does the buff disappear only on hit? Or is that also the case when you just summon a shade?

Reply:
only on hit